seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/FileSinkWriterWithTransaction.java

```diff
@@ -50,7 +50,7 @@ public class FileSinkWriterWithTransaction implements SinkWriter<SeaTunnelRow, F
 
     public FileSinkWriterWithTransaction(@NonNull SeaTunnelRowType seaTunnelRowTypeInfo,
                                          @NonNull Config pluginConfig,
-                                         @NonNull SinkWriter.Context context,
```

Comment:
I think it's best to keep class prefix, because `Context` was contained both `SourceReader` and `SourceWriter`, keep class prefix we can distinguish which class the `Context` comes from.